.github/workflows/test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         if: success() || failure() 
         uses: mikepenz/action-junit-report@v4
         with:
-          report_paths: "target/nextest/ci/junit.xml"
+          report_paths: "target/nextestgci/junit.xml"
```

Comment:
Just curious why this changed?

Reply:
Good catch, probably some ill replace command. I'll revert that.